ElrondNetwork/elrond-go
# Fix wrong genesis block save

- removed a wrong save of the genesis block in hash-nonce storage


Manual testing procedure:
- Start a testnet with nodes that do shuffling between shards. 
- Without the fix, we are able to see an error message containing the string `shard header (nonce-hash)` after a few epochs.
- With the fix, the error message will no longer be logged.
